### PR TITLE
internal audit no-brainers cleanup

### PR DESCRIPTION
**Pull request by RyanRHall**

### CODE REVIEW

File: pkg/v3/observation.go [hunk 1]
```diff
@@ -153,6 +154,19 @@ func validateCheckResult(r ocr2keepers.CheckResult, utg ocr2keepers.UpkeepTypeGe
 	return nil
 }
 
+func validateUpkeepProposal(p ocr2keepers.CoordinatedBlockProposal, utg ocr2keepers.UpkeepTypeGetter, wg ocr2keepers.WorkIDGenerator) error {
+	// No validation is done on Trigger.BlockNumber and Trigger.BlockHash because those
+	// get udpated with a coordinated quorum block
```
Comment (shileiwill) [inline pkg/v3/observation.go:159]: nit, typo from copy-pasta: udpated -> updated

File: pkg/v3/outcome.go [hunk 2]
```diff
@@ -32,11 +32,13 @@ const (
 
 // AutomationOutcome represents agreed upon state by the network, derived from
 // a collection of AutomationObservations with applied quorum thresholds
+// A byzQuorem # of nodes must agree that atleast f+1 nodes sent the same AgreedPerformable
```
Comment (shileiwill) [inline pkg/v3/outcome.go:35]: nit, typo: atleast -> at least

File: pkg/v3/plugin/coordinated_block_proposals.go [hunk 3]
```diff
@@ -41,6 +41,14 @@ func newCoordinatedBlockProposals(quorumBlockthreshold int, roundHistoryLimit in
...
 //     proposals and performables and then added to the outcome with quorum block
 //
+// Example of proposal coordination:
+// Round        1           2           3           4
+// Performables []          []          [A]         [B, C, E, F]
+// Proposals    [A, B]      [C, D]      [E, F]      []
+// ...                      [A, B]      [C, D]      []
+// ...                                  [B]         [D]
+// ...                                              []
+//
```
Comment (shileiwill) [inline pkg/v3/plugin/coordinated_block_proposals.go:51]: 👍

**Review by shileiwill — DISMISSED** [review]
@RyanRHall The PR looks great to me and it is cleaner and easy to follow. Thanks.

File: pkg/v3/plugin/coordinated_block_proposals.go (hunk 3 above)
Comment (infiloop2) [reply pkg/v3/plugin/coordinated_block_proposals.go:51]: ❤️

**Pull request merged**